TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -867,6 +867,20 @@ public static void CheckStatement(Node node)
                             break;
                         }
 
+                        if (receiver.Name.Length > 22)
+                        {
+                            DiagnosticUtils.AddError(node,
+                                "Pointer name '" + receiver.Name + "' is over 22 characters.");
+                            break;
```

Comment:
After reading your code and the code already in place, I think the `break` can be removed.
If a pointer variable is not declared correctly we want to have all error message at once.

So can you please remove the 3 `break` in this foreach ?